release-notes/opendistro-for-elasticsearch.anomaly-detection-kibana.release-notes-1.8.0.0.md

```diff
@@ -0,0 +1,10 @@
+## Open Distro for Elasticsearch 1.7.0 Release Notes
+Compatible with Kibana 7.7.0 and Open Distro for Elasticsearch 1.8.0.
+
+## Enhancements
```

Comment:
Will we add other changes to 1.8 ?

Reply:
yeah, this is just preliminary. will need to add additional PRs once we have all changes we want to include for this release